.pylintrc

```diff
@@ -0,0 +1,9 @@
+[MASTER]
+
+# Use multiple processes to speed up Pylint.
+jobs=2
```

Comment:
I found this through looking through pylint's long list of options, that you can see after using `--generate-rcfile`.